.travis.yml

```diff
@@ -1,13 +1,17 @@
+---
 sudo: false
 dist: trusty
 
 language: python
 python:
-  - 3.5
+  - 2.7
```

Comment:
Not sure what it happens as a consequence of this change here. Ubuntu Trusty (`dist: trusty` a couple of lines above) has Python 3 as a default. So I'm not sure which version is used with `python` and `pip` (my guess: Python 3), and not sure if a) is there a `python2` or `pip2`, and b) what Travis does when you set a Python version different from the Trusty default.

Saying we have two pairs of binaries, one for Python 2 (e.g. `python2` and `pip2`) and another for Python 3 (e.g. `python3` and `pip3`), I'd say the best shot would be to:

1. Install Ansible _globally_ (`pip2 install ansible`)
2. Create and activate a virtualenv with Python 3 in the `before_script` section (`python3 -m venv .venv && .venv/bin/activate`)
3. And from now on Jarbas will run within the virtualenv
4. Deactivate the virtualenv when Ansible is needed (`deactivate`)